azkaban-db/src/main/sql/create.validated_dependencies.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE validated_dependencies (
+  file_name         VARCHAR(100),
```

Comment:
Why different from `file_name` above? Why not `VARCHAR(128)`?

Reply:
`128` works fine too. Git diff is misleading - I didn't actually specify the `VARCHAR(128)` for `project_versions`. That was unmodified.